.circleci/config.yml

```diff
@@ -4,6 +4,9 @@ ignore_forks: &ignore_forks
     # Forked pull requests have CIRCLE_BRANCH set to pull/XXX
     ignore: /pull\/[0-9]+/
 
+env_integration_test: &env_integration_test
+  - SAUCE_USERNAME: lwc_ci
```

Comment:
Instead of hardcoding tokens here, is it possible to use an environment variable for it?
Even if the keys are not private, it would allow us to rotate the keys if needed without changing the code.